.github/workflows/update-oam-defs.yml

```diff
@@ -0,0 +1,94 @@
+name: Meshery Kuma Pattern Components Generator
+on:
+  push:
+    branches:
+      - "master"
+jobs:
+  check_kuma_version:
+    name: Check Kuma Version
+    runs-on: ubuntu-latest
+    outputs:
+      NEW_KUMA_VERSION: ${{ steps.glrt.outputs.release }}
+      CURRENT_KUMA_VERSION: ${{ env.CURRENT_KUMA_VERSION }}
+    steps:
+      - name: Get latest release tag
+        id: glrt
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: kumahq/kuma 
+          excludes: prerelease, draft
+      - name: Check out code
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Get CURRENT_KUMA_VERSION from .env
+        run: |
+          cat .github/.env > $GITHUB_ENV
```

Comment:
Hey @Shreyas220 consider changing this to https://github.com/meshery/meshery-osm/blob/master/.github/workflows/update-oam-defs.yml#L24-L31